libs/application/templates/driving-instructor-registrations/src/fields/ViewStudent/index.tsx

```diff
@@ -104,6 +107,8 @@ const ViewStudent = ({
   const studentRegistrations = student?.book
     ?.teachersAndLessons as Array<DrivingBookLesson>
 
+  const isUsersStudent = userNationalId === student?.book?.teacherNationalId
```

Comment:
Maybe easier to understand with variable name `isStudentOfUser` or `studentBelongsToUser`?